task.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# создаём каталог task с вложенными директориями
+# task
+#   dir1
+#   dir2
+#   dir3
+#       dir4
+mkdir -p ./task/dir1
+mkdir -p ./task/dir2
+mkdir -p ./task/dir3/dir4
+
+# изменяем текущую директорию на task
+cd ./task
```

Comment:
Слеши не обязательны
```suggestion
cd task
```